src/js/EditsDetail.jsx

```diff
@@ -5,7 +5,7 @@ var EditsDetail = React.createClass({
   propTypes: {
     details: React.PropTypes.array,
     primary: React.PropTypes.string,
-    setAppStatus: React.PropTypes.func
+    appStatus: React.PropTypes.object
```

Comment:
Same as above ...
Could we also define that the object should contain functions?
https://facebook.github.io/react/docs/reusable-components.html